src/components/faq/FaqHeading/FaqHeading.styled.ts

```diff
@@ -0,0 +1,15 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+
+export const Heading = styled.h2`
+  ${({ theme }) => css`
+    ${theme.fonts.kr.bold46};
+    flex-basis: 100%;
+    margin: 8.4rem 0 5.4rem 0;
```

Comment:
이렇게 바꿔줄 수도 있을거 같아요 :)
```suggestion
    margin: 8.4rem 0 5.4rem;
```